src/main/webapp/script.js

```diff
@@ -309,30 +309,49 @@ function addPoi(poi) {
 
 // Build and return a user-added POI HTML object.
 function buildPoiObject(poi) {
-  const formGroupContainer = document.createElement('div');
-  formGroupContainer.className = 'form-group';
-  formGroupContainer.style.display = 'block';
-
-  // Create poiInputButton, as well as function to remove the container upon 
-  // click. The submit button is also checked.
-  const poiInputButton = buildInput('button', poi + ' (click to remove)', 
-    'btn btn-secondary poi-input', poi);
-  poiInputButton.onclick = 
-    () => { formGroupContainer.remove(); checkSubmitButton() };
-
-  // Add the POI input button to the div container, and return the container.
-  formGroupContainer.appendChild(poiInputButton);
-  return formGroupContainer;
-}
+  // Create POI element, and remove the hover attributes.
+  const poiElement = document.createElement('button');
+  poiElement.className = 'btn-nohover btn-light-nohover';
+  poiElement.id = 'poi-element';
+  poiElement.type = 'button';
+
+  // Add span element with POI text.
+  const spanPoiText = document.createElement('span');
+  spanPoiText.className = 'poi-input';
+  spanPoiText.id = 'poi-element-text';
+  spanPoiText.innerText = poi;
+
+  // Add span element with spacing between POI text and trash can.
+  const spanPoiSpace = document.createElement('span');
+  spanPoiSpace.id = 'poi-element-space';
+  spanPoiSpace.innerHTML += '&nbsp;&nbsp;&nbsp;&nbsp;';
+
+  // Add trash icon to end of button, and add onclick function to icon.
+  const trashIconButton = document.createElement('i');
+  trashIconButton.id = 'poi-trash-icon';
+  trashIconButton.className = 'fa fa-trash';
+  trashIconButton.onclick = () => {
+    poiElement.remove();
+    checkSubmitButton();
+  };
+
+  // Add hover styling through mouseenter and mouseleave listeners.
+  trashIconButton.addEventListener('mouseenter', function(event) {
+    spanPoiText.style.textDecoration = 'line-through';
+    poiElement.style.backgroundColor = '#e2e6ea';
+    poiElement.style.borderColor = '#dae0e5';
+  });
+  trashIconButton.addEventListener('mouseleave', function(event) {
+    spanPoiText.style.textDecoration = 'none';
+    poiElement.style.backgroundColor = '#f8f9fa';
+    poiElement.style.borderColor = '#f8f9fa';
+  });
```

Comment:
I remember we went over this and we saw that there wasn't a huge change when hovering. Your call whether to leave or not just wanted to bring to your attention

Reply:
This hover effect is different than the one when hovering over the initial card. This is for hovering over the inputted POIs, and the text is crossed-through (thus `spanPoiText.style.textDecoration = 'line-through'`).